apps/web/pages/getting-started/[[...step]].tsx

```diff
@@ -112,6 +112,7 @@ const OnboardingPage = (props: IOnboardingPageProps) => {
             <StepCard>
               {currentStep === "user-settings" && <UserSettings user={user} nextStep={() => goToIndex(1)} />}
 
+              {/* TODO: filter out calendar apps that have external links */}
               {currentStep === "connected-calendar" && <ConnectedCalendars nextStep={() => goToIndex(2)} />}
 
```

Comment:
todo: we wanna filter out calendar apps with external links from getting started
- vimcal
- cron
- amie.so

Reply:
@hariombalhara will look into this